charts/hono/values.yaml

```diff
@@ -105,6 +105,14 @@ serviceType:
 messagingNetworkTypes:
 - "kafka"
 
+# useLegacyAmqpTraceContextFormat determines how OpenTelemetry trace context information
+# is stored in an AMQP 1.0 message sent to the AMQP Messaging Network. A "false" value here
+# configures the format used in previous Hono versions, while setting "true" chooses a more
+# generic format (compatible with e.g. Eclipse Ditto). Refer to
```

Comment:
this sounds wrong, shouldn't it be the other way around, i.e.
`useLegacyFormat=true` means that I want to use the old (legacy) format, right?

Reply:
oh, yes. It's fixed now.